tests/policy_as_code_test.go

```diff
@@ -76,21 +88,70 @@ func (pc *PolicyAsCodeSuite) SetupSuite() {
 	pc.informerfactory.WaitForCacheSync(pc.stopCh)
 }
 
-func (pc *PolicyAsCodeSuite) TestPolicyAsCode() {
+func (pc *PolicyAsCodeSuite) TestSaveAsCRUpdateDelete() {
 	policy := pc.createPolicyInCentral()
+
+	notifier := pc.createNotifierInCentral()
+	policy.Notifiers = []string{notifier.GetId()}
+
 	pc.policies = append(pc.policies, policy)
+	pc.notifier = notifier
 	k8sPolicy := pc.saveAsCustomResource(policy)
 	k8sPolicy = pc.createPolicyInK8s(k8sPolicy)
+
 	// Make sure the ID from Central is used to ensure controller didn't create a duplicate
 	pc.checkPolicyIsDeclarative(policy.Id)
 	pc.updateCRandObserveInCentral(k8sPolicy, policy.Id)
 	pc.deleteCRandObserveInCentral(k8sPolicy, policy.Id)
 }
 
+func (pc *PolicyAsCodeSuite) TestCreateCR() {
+	k8sPolicy := &v1alpha1.SecurityPolicy{
+		Spec: v1alpha1.SecurityPolicySpec{
+			PolicyName:      "test-policy-create",
+			Description:     "This is a description",
+			Categories:      []string{"Vulnerability Management"},
+			Severity:        storage.Severity_MEDIUM_SEVERITY.String(),
+			LifecycleStages: []v1alpha1.LifecycleStage{"DEPLOY"},
+			PolicySections: []v1alpha1.PolicySection{
+				{
+					SectionName: "Section 1",
+					PolicyGroups: []v1alpha1.PolicyGroup{
+						{
+							FieldName: "Days Since CVE Was First Discovered In Image",
+							Values: []v1alpha1.PolicyValue{
+								{
+									Value: "5",
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+	k8sPolicy.SetName("test-policy-cr")
```

Comment:
The policy CR is not getting cleaned up because it needs the label.  See how the delete is happening in the tear down function.
```suggestion
	k8sPolicy.SetName("test-policy-cr")
	k8sPolicy.SetLabels(map[string]string{
		"test": "policy-as-code",
	})

```